test/spec/functional/chaiPromises.js

```diff
@@ -2,16 +2,14 @@ describe('chai-as-promised', function() {
 
     /**
      *  Remove the `should` global on Object.prototype to allow chai.should for these tests,
-     *  and set up chai, chai-as-promised and chai.should properly
+     *  and set up chai and chai.should properly
      */
 
     require('should').noConflict();
 
     var chai = require('chai');
-    var chaiAsPromised = require('chai-as-promised');
 
     chai.Should();
-    chai.use(chaiAsPromised);
```

Comment:
Oh, wait...  Why did I change this...  O_o

I had to change something to I could use .rejectedWith(), but I did that in
the same file as defines 'setup()'.
On Sep 11, 2015 18:37, "Christian Bromann" <notifications@github.com> wrote:

> In test/spec/functional/chaiPromises.js
> <https://github.com/webdriverio/webdriverio/pull/779#discussion_r39325363>
> :
>
> >
> >      chai.Should();
> > -    chai.use(chaiAsPromised);
>
> What was the error? This should actually not affect your test as it is a
> functional test.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webdriverio/webdriverio/pull/779/files#r39325363>.
>